### PR TITLE
Returning correct error when user is unable to change password

### PR DESCRIPTION
**Pull request by Alevsk**
## When password change fails because of permissions/policies

Screen Shot 2021-03-09 at 00 07 31

## When password change fails because of incorrect current password

Screen Shot 2021-03-09 at 00 07 04

## Change password is enabled

Screen Shot 2021-03-09 at 15 44 10

## Change password is disabled

Screen Shot 2021-03-09 at 15 43 59

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

**Review by kannappanr — DISMISSED** [review]
LGTM

**Review by bexsoft — APPROVED** [review]

File: restapi/error.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ var (
 	errInvalidLicense                     = errors.New("invalid license key")
 	errLicenseNotFound                    = errors.New("license not found")
 	errAvoidSelfAccountDelete             = errors.New("logged in user cannot be deleted by itself")
+	errAccessDenied                       = errors.New("Access Denied")
```
Comment (bexsoft) [inline restapi/error.go:38]: Please make it lower case
```suggestion
	errAccessDenied                       = errors.New("access Denied")
```

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please fix

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by dvaldivia — DISMISSED** [review]

**Comment by cesnietor** [comment]
might need to remake assets due to previous merge

**Review by bexsoft — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
on hold

**Comment by harshavardhana** [comment]
Why not disable the UI element entirely?

**Comment by Alevsk** [comment]
> Why not disable the UI element entirely?

good idea, adding support to enable/disable specific UI elements and not only options at the menu, with this we will improve UX for console users

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please change that line, beside of that, looks ok

**Review by bexsoft — APPROVED** [review]

**Review by cesnietor — APPROVED** [review]

**Pull request merged**